UITests/Global.swift

```diff
@@ -195,7 +195,7 @@ extension KIFUITestActor {
 
         webView.evaluateJavaScript("typeof KIFHelper") { result, _ in
             if result as! String == "undefined" {
-                let bundle = Bundle(for: NavigationTests.self)
```

Comment:
The purpose of using `NavigationTests.self` here is to get a reference to the unit test's `Bundle` instead of the app's Bundle `Bundle.main`. This is so we can reference resources that are bundled in from the tests instead of the resources bundled with the app. IIRC the test bundle contains the fake .html and .json files we use.

In short, as long as the class referenced in the `Bundle(for:)` call belongs to the test target this should be good!